client/my-sites/checkout/composite-checkout/hooks/use-create-payment-methods/index.ts

```diff
@@ -416,5 +516,5 @@ export default function useCreatePaymentMethods( {
 		epsMethod,
 		wechatMethod,
 		bancontactMethod,
-	].filter( Boolean );
+	].filter( doesValueExist );
```

Comment:
`doesValueExist` (poorly named; it should be something like `isValueNotFalsy`) is a TypeScript-safe way to filter out null values.

Reply:
I think `isTruthy` is the most specific description of how it works (`!!`)